js/app.js

```diff
@@ -0,0 +1,96 @@
+(function() {
+var FizzBuzz = (function() {  
+// scoping FizzBuzz in case it needs to be used elsewhere
+  var FizzBuzz = function() {}; 
+  // defining FizzBuzz as a function, but can be empty b/c 
+  // all the functionality is in the other functions below
+  
+  function doFizzBuzz(start,stop) {  
```

Comment:
Make your code a little more readable and put a space after the comma in arguments lists.